### PR TITLE
Add Notify Notifications Channel

### PR DESCRIPTION
**Pull request by nickbruylandts**

### CODE REVIEW

**Comment by atymic** [comment]
Hi!

Are you still open to creating and maintaining this channel?

If so, could you update it to support laravel 5.5-6.0 please?

Thanks!

File: src/NotifyMessageResource.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class NotifyMessageResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     * @param \Illuminate\Http\Request $request
+     * @return array
+     */
+    public function toArray($request)
```
Comment (atymic) [inline src/NotifyMessageResource.php]: This could just be on the class (just implement `JsonSerializable`)

File: src/NotifyServiceProvider.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+{
+    /**
+     * Bootstrap the application services.
+     */
+    public function boot()
+    {
+        $this->app->when(NotifyChannel::class)
+            ->needs(NotifyClient::class)
+            ->give(function () {
+                $config = config('services.notify');
```
Comment (atymic) [inline src/NotifyServiceProvider.php]: Can pass this to the client constructor as above

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+language: php
+
+php:
+  - 7.3
```
Comment (atymic) [inline .travis.yml]: 7.2 as well please

File: README.md [hunk 4]
```diff
@@ -1,17 +1,179 @@
...
+[![StyleCI](https://styleci.io/repos/:style_ci_id/shield)](https://styleci.io/repos/:style_ci_id)
+[![SensioLabsInsight](https://img.shields.io/sensiolabs/i/:sensio_labs_id.svg?style=flat-square)](https://insight.sensiolabs.com/projects/:sensio_labs_id)
+[![Quality Score](https://img.shields.io/scrutinizer/g/laravel-notification-channels/notify.svg?style=flat-square)](https://scrutinizer-ci.com/g/laravel-notification-channels/notify)
+[![Code Coverage](https://img.shields.io/scrutinizer/coverage/g/laravel-notification-channels/notify/master.svg?style=flat-square)](https://scrutinizer-ci.com/g/laravel-notification-channels/notify/?branch=master)
+[![Total Downloads](https://img.shields.io/packagist/dt/laravel-notification-channels/notify.svg?style=flat-square)](https://packagist.org/packages/laravel-notification-channels/notify)
 
-### I'm working on a new channel
-Please create an issue for it if it does not already exist, then PR you code for review.
+This package makes it easy to send notifications using [Notify](https://app.notify.eu) with Laravel 5.6+ & 6.x
```
Comment (atymic) [inline README.md]: Can we link to the homepage instead of the login page please

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,55 @@
+{
+    "name": "laravel-notification-channels/notify",
```
Comment (atymic) [inline composer.json]: i'm thinking we should maybe call this notify-eu. Thoughts?

File: src/NotifyClient.php [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+        $this->client = $client;
+        $this->config = $config;
+    }
+
+    /**
+     * @param NotifyMessage $message
+     */
+    public function validateMessage(NotifyMessage $message)
+    {
+        if (empty($message->getClientId()) or empty($message->getSecret())) {
```
Comment (atymic) [inline src/NotifyClient.php]:
```suggestion
        if (empty($message->getClientId()) || empty($message->getSecret())) {
```

File: src/NotifyMessage.php [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+    private $params;
+    private $notificationType;
+    private $language;
+    private $transport;
+    private $to;
+    private $cc;
+    private $bcc;
+
+    /**
+     * Message constructor.
```
Comment (atymic) [inline src/NotifyMessage.php]: Can remove this docblock

**Comment by atymic** [comment]
Just a few minor things, then i'll create the repo :)

**Comment by nickbruylandts** [comment]
@atymic any chance you have some time to review the changes?
We would like to use the package in a new project.

Thx!

**Comment by nickbruylandts** [comment]
@atymic Any chance of reviewing the changes?

**Pull request closed without merging**